src/components/BookList.js

```diff
@@ -1,14 +1,20 @@
-import { useSelector } from 'react-redux';
+import { useSelector, useDispatch } from 'react-redux';
+import { useEffect } from 'react';
 import Book from './Book';
 import Form from './Form';
+import { getBookList } from '../redux/books/books';
 
 const BookList = () => {
+  const dispatch = useDispatch();
+  useEffect(() => {
+    dispatch(getBookList());
+  }, []);
```

Comment:
- [ ] Nice job so far, however, books are not being displayed on your page after page reload, Kindly fix this